src/devices/sound/disc_wav.hxx

```diff
@@ -746,7 +746,7 @@ DISCRETE_STEP(dss_op_amp_osc)
 			}
 			/* we need to mix any bias and all modulation voltages together. */
 			v = DSS_OP_AMP_OSC__VMOD1 - OP_AMP_NORTON_VBE;
-			if (v < 0) v = 0;
+			// if (v < 0) v = 0; /* no clipping. this voltage can be minus, for charging +/- to "C" */
```

Comment:
Go ahead and just remove the original if statement here.  The comment you added is fine, but it should be C++ // style.  The explanation sounds reasonable to me with my limited knowledge of circuits, but @couriersud if you're around and could comment that would be great.

Reply:
Thanks for your point out! fixed it.
If approvable, please could you re-open the PR? or should I create a new PR?

Reply:
Opened PR #11287 
If you don't mind, could you please review it?